doc/monitoring-stack-deployment/datasource-v5.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: grafana.integreatly.org/v1beta1
+kind: GrafanaDatasource
+metadata:
+  name: prometheus
+spec:
+  name: middleware
+  instanceSelector:
```

Comment:
Seems instanceSelector can be deleted from GrafanaDatasource, suppose it's not the place, and it also not in use.
You set it directly in Dashboard creation (that correct).

Reply:
Thanks, instanceSelector is a required field for Grafana v5 datasource

Reply:
Thanks for reply.  As seems to me instanceSelector is required in Dashboard (you do it), but datasource is not a place.  I will test it, but  if you have reference to documentation , can you please send. Thanks

Reply:
Yes, I see: 
oc apply -f datasource.yaml
The GrafanaDatasource "prometheus" is invalid: 
* spec.instanceSelector: Required value

Thank you.